go.mod

```diff
@@ -8,21 +8,21 @@ require (
 	github.com/buger/goterm v1.0.4
 	github.com/checkpoint-restore/checkpointctl v0.0.0-20220321135231-33f4a66335f0
 	github.com/checkpoint-restore/go-criu/v6 v6.3.0
-	github.com/container-orchestrated-devices/container-device-interface v0.5.3
+	github.com/container-orchestrated-devices/container-device-interface v0.5.4
 	github.com/containernetworking/cni v1.1.2
 	github.com/containernetworking/plugins v1.2.0
 	github.com/containers/buildah v1.29.3
-	github.com/containers/common v0.51.3
+	github.com/containers/common v0.52.0
```

Comment:
This aligns with CRI-O release `1.25`. Newer releases are already on version `0.53.0` or newer.